usage/installer/src/main/assembly/files/blueprints/seaclouds-on-byon.yaml

```diff
@@ -19,14 +19,14 @@ services:
   - serviceType: "classpath://brooklyn/entity/brooklynnode/brooklyn-node.yaml"
     id: deployer
     name: SeaClouds Deployer
-    install.version: 0.8.0-SNAPSHOT
     managementUser: admin
     managementPassword: p4ssw0rd
     brooklynLocalPropertiesContents: |
       brooklyn.webconsole.security.users=admin
       brooklyn.webconsole.security.user.admin.password=p4ssw0rd
     brooklyn.config:
-      brooklynnode.download.archive.subpath: brooklyn-dist-0.8.0-SNAPSHOT/
+      version: 0.7.0-incubating
```

Comment:
I think this `ConfigKey` could be removed because `0.7.0-incubating` defines this version as [default value](https://github.com/apache/incubator-brooklyn/blob/0.7.0-incubating/software/base/src/main/java/brooklyn/entity/brooklynnode/BrooklynNode.java#L68).